theme/less/_header.less

```diff
@@ -1,13 +1,12 @@
 header
 {
-  display: flex;
-  margin: 12px 30px; 
+  margin: 25px 30px; 
```

Comment:
I think there's need to convert pixels to em's.

Reply:
- Let's keep pixels for margins and paddings for now, regarding the `font-size` - no strong preference.
- Also it seems like we are overwriting `rem` font-sizes with pixels in some places - let's fix it.